From: Michael Biebl <biebl@debian.org>
Date: Sun, 27 Apr 2014 18:10:58 +0200
Subject: Adjust systemd-user pam config file for Debian

This pam config file is used by libpam-systemd/systemd-logind when
launching systemd user instances.
---
 src/login/systemd-user.m4 | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/src/login/systemd-user.m4 b/src/login/systemd-user.m4
index 7933508..6b21de6 100644
--- a/src/login/systemd-user.m4
+++ b/src/login/systemd-user.m4
@@ -2,10 +2,6 @@
 #
 # Used by systemd --user instances.
 
-account  include system-auth
-
-m4_ifdef(`HAVE_SELINUX',
-session  required pam_selinux.so close
-session  required pam_selinux.so nottys open
-)m4_dnl
-session  include system-auth
+@include common-account
+@include common-session-noninteractive
+session optional pam_systemd.so
